src/main/java/uk/gov/hmcts/darts/task/runner/SoftDelete.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.darts.task.runner;
+
+import uk.gov.hmcts.darts.common.entity.UserAccountEntity;
+
+import java.time.OffsetDateTime;
+
+public interface SoftDelete {
```

Comment:
Java doc here would be good, highlighting how this interface is envisaged to be used i.e. in the context of an Entity.